.cargo/audit.toml

```diff
@@ -0,0 +1,2 @@
+[advisories]
+ignore = ["RUSTSEC-2021-0137"]
```

Comment:
If we got rid of libsodium and sodiumoxide, we can remove this I guess?

Reply:
we still have proteus 1.x in the tree, so this is still required.